app/controllers/api/attempts/events_controller.rb

```diff
@@ -76,6 +70,25 @@ def redis_client
       def request_token
         @request_token ||= AttemptsApi::RequestTokenValidator.new(request.authorization)
       end
+
+      def validate_params
+        if poll_params[:maxEvents].present? && !poll_params[:maxEvents].to_i.between?(1, 1000)
```

Comment:
We should probably consider creating a Form if we add more or more complex validations

Reply:
are there others you think we should add? or is this ok for now?

Reply:
I think ok for now, this fixes the important issue where we'd try to fetch infinite events if we received a request for it.